### PR TITLE
Ava/initial setup

### PR DESCRIPTION
**Pull request by AvaDeLaCruz**
Started website setup for landing page:
- changed color scheme to match the provided color palette
- added placeholder text to represent section content
- changed logo and flower colors

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import Head from "next/head";
+// import Header from './components/Header'
 
 import styled from "styled-components";
 
 import HackSC from "../assets/hacksc.svg";
 import WhiteFlower from "../assets/white_flower.svg";
 
+const Ava = () => {
```
Comment (WilhelmWillie) [inline pages/index.js]: lol nice

File: pages/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function Header(props) {
```
Comment (WilhelmWillie) [inline pages/components/Header.js]: before we merge tho let's remove unused code (unless there's plans to use this in the future, if so, id leave a `// TODO: add description here` comment

File: pages/index.js [hunk 3]
```diff
@@ -35,14 +39,78 @@ const Landing = () => {
...
+                        20
+                        20
+              to the right of the HackSC*/}
+              <TwentyRight>2020</TwentyRight>
+              <LogoText src={HackSC} alt="HackSC" />
+              <HeaderTitle>JAN 31, 2020 - FEB 2, 2020</HeaderTitle>
+              <HeaderSubtitle>LOS ANGELES, CALIFORNIA</HeaderSubtitle>
+            </TitleContainer>
+
+            <Links>{/* <Button>
```
Comment (WilhelmWillie) [inline pages/index.js]: same thing, we'll prob want to comment this out and add a TODO comment (just to keep track of things we want to come back to)

File: pages/index.js (hunk 1 above)
Comment (AvaDeLaCruz) [reply pages/index.js]: lol i was trying to test how to make a component and nothing was showing up and then i realized i needed to `npm run dev`

**Pull request closed without merging**